Update iot-hub-devguide-messages-read-custom.md

katriendg opened the pull request:
Added a note about the fact that messages are no longer delivered to the built-in endpoint once you create routes. However, there is an option to add a route to the default built-in endpoint. Something that is not clear in any of the documentation or even stands out in the portal.

azurecla commented:
Hi __@mattridgway__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

katriendg committed:
Update iot-hub-devguide-messages-read-custom.md

Added a note about the fact that messages are no longer delivered to the built-in endpoint once you create routes. However, there is an option to add a route to the default built-in endpoint. Something that is not clear in any of the documentation or even stands out in the portal.

dominicbetts commented on articles/iot-hub/iot-hub-devguide-messages-read-custom.md:
```diff
@@ -32,6 +32,10 @@ A single message may match the condition on multiple routing rules, in which cas
 An IoT hub has a default [built-in endpoint][lnk-built-in]. You can create custom endpoints to route messages to by linking other services in your subscription to the hub. IoT Hub currently supports Azure Storage containers, Event Hubs, Service Bus queues, and Service Bus topics as custom endpoints.
 
 > [!NOTE]
+> When you start using routing with custom endpoints, messages will no longer be delivered to the built-in endpoint unless they don't match any rules. In order to keep message delivery to the  built-in endpoint you can add a route and choose the endpoint named 'events'. 
```
Can I suggest a slightly different wording?
When you use routing and custom endpoints, messages are only delivered to the built-in endpoint if they don't match any rules. To deliver messages to the built-in endpoint as well as to a custom endpoint, add a route that sends messages to the **events** endpoint.

dominicbetts replied:
@katriendg - thanks again for your contribution. I've made a change directly to the content that should be published later today. Therefore I'm closing this PR.

dominicbetts reviewed (changes requested):
@katriendg many thanks for this proposed change - can you make some minor changes to the wording as I suggested. Then we can merge your changes.
